src/errors.ts

```diff
@@ -68,36 +70,24 @@ export class IntegrityConstraintViolationError extends SlonikError {
 
 export class NotNullIntegrityConstraintViolationError extends IntegrityConstraintViolationError {
   constructor (error: Error, constraint: string) {
-    super(error, constraint);
-
-    this.originalError = error;
-    this.message = 'Query violates a not NULL integrity constraint.';
+    super(error, constraint, 'Query violates a not NULL integrity constraint');
   }
 }
 
 export class ForeignKeyIntegrityConstraintViolationError extends IntegrityConstraintViolationError {
   constructor (error: Error, constraint: string) {
-    super(error, constraint);
-
-    this.originalError = error;
-    this.message = 'Query violates a foreign key integrity constraint.';
+    super(error, constraint, 'Query violates a foreign key integrity constraint');
   }
 }
 
 export class UniqueIntegrityConstraintViolationError extends IntegrityConstraintViolationError {
   constructor (error: Error, constraint: string) {
-    super(error, constraint);
-
-    this.originalError = error;
-    this.message = 'Query violates a unique integrity constraint.';
+    super(error, constraint, 'Query violates a unique integrity constraint');
   }
 }
 
 export class CheckIntegrityConstraintViolationError extends IntegrityConstraintViolationError {
   constructor (error: Error, constraint: string) {
-    super(error, constraint);
-
-    this.originalError = error;
-    this.message = 'Query violates a check integrity constraint.';
```

Comment:
It is generally a bad idea to make minor stylistic changes such as removing a dot from the end of a sentence, because these are subjective and do not directly relate to the issue that the PR is trying to address.

Reply:
I removed it because when concatenated with the original message and the colon, it looks strange: `Query violates unique constraint.: duplicate key value violates unique constraint "person_pkey"`.

But I agree with you about keeping changes minimal, so I'll get rid of the colon instead.